apps/docs/content/guides/auth/third-party/firebase-auth.mdx

```diff
@@ -69,6 +69,46 @@ let supabase = SupabaseClient(
 
 </TabPanel>
 
+<TabPanel id="kotlin" label="Kotlin (Android)">
+
+Create a Supabase client with the `accessToken` function that returns the Firebase Auth JWT of the current user. This code uses the [official Firebase SDK](https://firebase.google.com/docs/auth/android/start) for Android.
+
+```kotlin
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+
+val supabase = createSupabaseClient(
+    "https://<supabase-project>.supabase.co",
+    "SUPABASE_ANON_KEY"
+) {
+    accessToken = {
+        Firebase.auth.currentUser?.getIdToken(false)?.await()?.token
+    }
+}
+```
+
+</TabPanel>
+
+<TabPanel id="kotlinmp" label="Kotlin (Multiplatform)">
```

Comment:
Wasn't sure about this, should we use two tabs for Kotlin or put both samples into one?

Reply:
two tabs is good!